gtdynamics/jumpingrobot/src/jr_visualizer.py

```diff
@@ -124,39 +127,62 @@ def make_plot(values, jr, num_steps):
             qs_dict[name].append(gtd.JointAngle(values, j, k))
             vs_dict[name].append(gtd.JointVel(values, j, k))
             torques_dict[name].append(gtd.Torque(values, j, k))
-            pressures_dict[name].append(values.atDouble(Actuator.PressureKey(j, k)))
+            pressures_dict[name].append(
+                values.atDouble(Actuator.PressureKey(j, k)))
             masses_dict[name].append(values.atDouble(Actuator.MassKey(j, k)))
-            mdots_dict[name].append(values.atDouble(Actuator.MassRateActualKey(j, k)))
-            contractions_dict[name].append(values.atDouble(Actuator.ContractionKey(j, k)))
+            mdots_dict[name].append(
+                values.atDouble(Actuator.MassRateActualKey(j, k)))
+            contractions_dict[name].append(
+                values.atDouble(Actuator.ContractionKey(j, k)))
```

Comment:
Some of the changes in this file look like just different line lengths in linter/autoformatter settings, and I think the one-line versions are cleaner and not that long, maybe allow what I assume is the 100-char width in this file?

Reply:
Is line-length=100 the default line length we accept? This is the same setting as what the other python files are formatted with, so in the interest of consistency, I am genuinely wondering what the correct approach is here.

Reply:
oh I thought maybe you set to 80-char, if this is 100 then nvm :)